tests/unit/lib/reporters/JsonCoverage.js

```diff
@@ -0,0 +1,64 @@
+define([
+	'intern!object',
+	'intern/chai!assert',
+	'dojo/node!istanbul/lib/collector',
+	'dojo/node!istanbul/lib/report/json',
+	'dojo/node!fs',
+	'./support/mocks',
+	'../../../../lib/reporters/JsonCoverage'
+], function (registerSuite, assert, Collector, Reporter, fs, mock, JsonCoverage) {
+	var sessionId = 'foo';
+
+	registerSuite({
+		name: 'intern/lib/reporters/JsonCoverage',
+
+		coverage: function () {
```

Comment:
I guess the same could be said for the [Lcov Reporter Tests](https://github.com/theintern/intern/blob/master/tests/unit/lib/reporters/Lcov.js) as well.  I was just following the pattern set out in the project.  As I assume whomever wrote the Lcov unit tests, they also felt that unit tests should only check the implementation details of the code and that introspecting the output would actually be testing Istanbul...  Maybe the author of the Lcov unit wouldn't mind commenting on their logic...